Modules/vc-module-pricing/VirtoCommerce.PricingModule.Web/Scripts/blades/assignment-list.js

```diff
@@ -7,6 +7,8 @@ angular.module('virtoCommerce.pricingModule')
 
             var exportDataRequest = {
                 exportTypeName: 'PricelistAssignment',
+                isTabularExportSupported: true,
+                defaultProvider: 'JsonExportProvider',
```

Comment:
Let's add this
            isAllSelected: true,
And remove additional set in line 162